.github/workflows/ecosystem-ci-trigger.yml

```diff
@@ -0,0 +1,92 @@
+name: ecosystem-ci trigger
+
+on:
+  issue_comment:
+    types: [created]
+
+jobs:
+  trigger:
+    runs-on: ubuntu-latest
+    if: github.event.issue.pull_request && startsWith(github.event.comment.body, '/ecosystem-ci run')
```

Comment:
I would use `!ecosystem-ci`, because all our magic comments use `!` at the beginning